ide-common/src/main/java/org/digma/intellij/plugin/insights/view/GroupListViewItemBuilder.java

```diff
@@ -17,19 +17,21 @@
 public class GroupListViewItemBuilder<T extends CodeObjectInsight> implements ListViewItemBuilder<T> {
 
     private final InsightGroupType insightGroupType;
-    private final Function<T, String> groupByFunction;
+    private final Function<T, String> routeFunction;
+    private final Function<T, String> endpointSpan;
```

Comment:
why do we need 2 entries?
there was a generic one, named `groupByFunction`
also routeFunction is just for HTTP, but we have also GRPC

Reply:
yes